config-examples/peer-b-config.toml

```diff
@@ -0,0 +1,18 @@
+public_key = "peer-b-public-key"
+secret_key = "peer-b-secret-key"
+listen = ["[::]:10002"]
+verbosity = "Quiet"
+
+[[peers]]
```

Comment:
Can you add a second peer in an example?

Reply:
I never tried _yet_ :smile: